api/src/main/resources/workbench-api.yaml

```diff
@@ -5286,6 +5287,12 @@ definitions:
       eraCommonsCompletionTime:
         type: integer
         format: int64
+      rasLinkLoginGovBypassTime:
```

Comment:
Are these being populated now? I don't see the change in this PR - is it happening automatically in a mapper somewhere?

Reply:
I did update the query and a new getAasLinkLoginGovBypassTime() in userDao's `DbAdminTableUser`. And it seems to be working.